app/konnect/org-management/audit-logging/reference.md

```diff
@@ -0,0 +1,114 @@
+
+## Authentication logs
+
+Example log entry:
+
+```
+Jan 16 18:25:28 
+konghq.com CEF:0|ExampleOrg|Konnect|1.0|AUTHENTICATION_TYPE_PAT|AUTHENTICATION_OUTCOME_SUCCESS|0|rt=3958q3097698 
+src=127.0.0.1 
+request=/api/v1/personal-access-tokens/introspect 
+success=true 
+org_id=b065b594-6afc-4658-9101-5d9cf3f36b7b 
+principal_id=87655c36-8d63-48fe-9a1e-53b28dfbc19b 
+trace_id=3895213347334635099 
+user_agent=grpc-go/1.51.0
+```
+
+Each authentication log entry contains the following:
+
+Property | Description
+---------|-------------
+Timestamp | Time and date of the event in UTC.
+`AUTHENTICATION_TYPE` | Can be one of the following: <br> - `AUTHENTICATION_TYPE_BASIC`: basic email and password authentication <br> - `AUTHENTICATION_TYPE_SSO`: authentication with single sign-on (SSO) <br> - `AUTHENTICATION_TYPE_PAT`: authentication with a personal access token
+`AUTHENTICATION_OUTCOME` | Can be one of the following: <br> - `AUTHENTICATION_OUTCOME_SUCCESS`: authentication is successful<br> - `AUTHENTICATION_OUTCOME_NOT_FOUND`: user was not found<br> - `AUTHENTICATION_OUTCOME_INVALID_PASSWORD`: invalid password specified <br> - `AUTHENTICATION_OUTCOME_LOCKED`: user account is locked<br> - `AUTHENTICATION_OUTCOME_DISABLED`: user account has been disabled
+`org_id` | The originating organization ID.
+`principal_id` | The user ID of the user that performed the action.
+`rt` | ?
+`src` | The IP address of the request originator.
+`user_agent` | The user agent of the request: application, operating system, vendor, and version.
+`success` | `true` or `false`, depending on whether authentication was successful or not.
+
+
+## Authorization logs
+
+Example log entry:
+
+```
+Jan 16 00:25:40 
+konghq.com CEF:0|ExampleOrg|Konnect|1.0|konnect|Authz.portals|1|rt=16738287345642 
+src=127.0.0.6 
+action=retrieve 
+granted=true 
+org_id=b065b594-6afc-4658-9101-5d9cf3f36b7b 
+principal_id=87655c36-8d63-48fe-9a1e-53b28dfbc19b 
+actor_id= 
+trace_id=8809518331550410226 
+user_agent=grpc-node/1.24.11 grpc-c/8.0.0 (linux; chttp2; ganges)
+```
+
+Each authorization log entry contains the following:
+
+Property | Description
+---------|-------------
+Timestamp | Time and date of the event in UTC.
+`action` | The type of action the user performed on the resource.
+`granted` | Boolean indicating whether the authorization was granted or not.
+`org_id` | The originating organization ID.
+`principal_id` | The user ID of the user that performed the action.
+`actor_id` | If using impersonation, the ID of the person doing the impersonation. Otherwise, this field is empty.
+`trace_id` | The trace ID of the request.
+`caller_ip` | IP address of the user agent making the request.
+`user_agent` | The user agent of the request: application, operating system, vendor, and version.
+`service_name` | The name of the component within Konnect. For example, `portals` or `runtimegroups`.
+
+## Actions logged by Konnect
```

Comment:
thought: I'm worried that if we enumerate every action logged, it will be hard to keep this document up-to-date. We log reads and writes to all entities in Konnect. As we add new entities, we'll need to update this list.

Reply:
Hmm. Yeah, that seems like an unnecessary maintenance burden. It should be sufficient to say "we log all reads and writes to all entities in Konnect".